{
  "error": "Task instance 'split_event_groups' raised ValueError('Value grouper '%1234' is neither a column nor an index in the DataFrame')",
  "result": null,
  "trace": "str"
}
